### PR TITLE
Removing source-maps scss

### PR DESCRIPTION
**Pull request by iondrimba**
**CHANGELOG** :memo:

* Removed source-maps generation

### CODE REVIEW

**Comment by iondrimba** [comment]
[(0.5) Bug: precompile com algumas URLs quebradas (de fontes e .map)](https://trello.com/c/be4CsS4q/326-0-5-bug-precompile-com-algumas-urls-quebradas-de-fontes-e-map)

File: bin/build/prod/css.js [hunk 1]
```diff
@@ -5,18 +5,16 @@ const shelljs = require('shelljs');
 const sourceDir = "./src/scss";
 const outputDir = "./dist";
 const compiledFile = `${outputDir}/gaiden.css`;
-const browserSupport = "> 0.1% in BR, not ie 8";
```
Comment (bsoares) [inline bin/build/prod/css.js]: Why do you remove this?

Comment (iondrimba) [inline bin/build/prod/css.js]: @bsoares We are using this config from package.json now

**Comment by iondrimba** [comment]
[(0.5) Bug: precompile com algumas URLs quebradas (de fontes e .map)](https://trello.com/c/be4CsS4q/326-0-5-bug-precompile-com-algumas-urls-quebradas-de-fontes-e-map)

**Pull request merged**